gmime/part.go

```diff
@@ -151,9 +151,32 @@ func (p *Part) SetHeader(name string, value string) {
 	C.g_mime_header_list_set(headers, cName, cValue, cCharset)
 }
 
-// Headers gives you all headers for part
-func (p *Part) Headers() textproto.MIMEHeader {
-	return nil
+// GetHeaders gives you all headers for part
+func (p *Part) GetHeaders() textproto.MIMEHeader {
```

Comment:
usually you just don't name things `Get...()` in go

Reply:
I figured it plays nicely with GetHeader & GetHeaders 🤷